h/migrations/versions/3359d7d8ec29_remove_user_last_login_date_default_values.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
```

Comment:
We should remove this from [the migration script template](https://github.com/hypothesis/h/blob/master/h/migrations/script.py.mako) as it's not needed in Python 3